koch.nim

```diff
@@ -470,7 +470,7 @@ proc temp(args: string) =
   # commit.
   let (bootArgs, programArgs) = splitArgs(args)
   let nimexec = findNim()
-  exec(nimexec & " c -d:debug " & bootArgs & " compiler" / "nim", 125)
+  exec(nimexec & " c -d:debug --debugger:native " & bootArgs & " compiler" / "nim", 125)
```

Comment:
Why was this added?

Reply:
so that the nim_temp build has debug information. Debug information is necessary to use debuggers like gdb. Well to be precise one can debug without debug information, but then then you debug in disassembled binary instead of seeing the actual source.

Reply:
Just try it, and run ``gdb nim_temp``.  There are two possible outputs:

```
Reading symbols from nim_temp...done.
```

```
Reading symbols from nim_temp...(no debugging symbols found)...done.
```